.circleci/config.yml

```diff
@@ -98,12 +99,16 @@ commands:
 
             echo "Test Domain is" $TEST_DOMAIN
 
+            npx @adobe/cci-test-aggregator ${pipeline_url}
```

Comment:
where is this ?

Reply:
https://github.com/adobe/cci-test-aggregator